README.md

```diff
@@ -323,8 +323,8 @@ This field is present if the `status` field is `committed` or `finalized`, the `
 The value is the account address of the recipient of the transfer.
 
 #### `amount` (optional)
-This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`, and the transaction is a simple transfer.
-The value is a number representing the amount transferred from the sender to the recipient.
+This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`, and the transaction is a simple transfer or a `InitContract`.
+In case of a simple transfer, the value is a number representing the amount transferred from the sender to the recipient. In the case of a `InitContract`, the value is a number representing the amount the contract is initialized with.
```

Comment:
... and this is the amount that is transferred from the sender account to the contract.